Annotato/Annotato/Views/Document/Edit/Annotation/AnnotationView.swift

```diff
@@ -125,6 +129,26 @@ class AnnotationView: UIView {
                 }
             }
         }.store(in: &cancellables)
+
+        viewModel.$isResolving.sink { [weak self] isResolving in
+            guard !isResolving else {
+                return
+            }
+            guard let self = self else {
+                return
+            }
+            if self.viewModel.resolveBySave {
+                let mergeConflictsHeight = self.mergeConflictsPalette.height
+                self.mergeConflictsPalette.resetDimensions()
+                self.mergeConflictsPalette.removeFromSuperview()
+
+                self.palette.translateUp(by: mergeConflictsHeight)
```

Comment:
Continuing the above, if that happened, then annotationViewModel can tell the palette to move up also? Same for the scroll.

Reply:
Same as the previous comment
Using subscribers also right? Palette will have to listen for changes in the view model's frame, then reassign the UIView's frame accordingly

Currently reassigning the frame is how I am shifitng the views